scripts/ci/runners/sync_authors.py

```diff
@@ -42,15 +42,8 @@
 req = requests.get(AUTHORS)
 author_list = toml.loads(req.text)
 
-author_set = set()
-for membership in author_list:
-    author_set.update([author for author in author_list[membership]])
-
-authors = ""
-for author in sorted(author_set):
-    authors += f'            "{author}",\n'
-
-authors = authors[:-2]
+author_set = set().union(*author_list.values())
+authors = ",\n".join(f'            "{author}"' for author in author_set)
```

Comment:
```suggestion
authors = ",\n".join(f'            "{author}"' for author in sorted(set(author_list.values())))
```

Reply:
Looks like https://github.com/apache/airflow-ci-infra/pull/58 removed that authors file anyway.

Reply:
Indeed, I opened #33350 to remove the file, could you revert this change?